src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -247,22 +261,327 @@ def get(self, request: Request, organization) -> Response:
         return response
 
 
+class OrganizationAlertRuleIndexPostSerializer(serializers.Serializer):
+    name = serializers.CharField(max_length=64, help_text="The name for the rule.")
+    aggregate = serializers.CharField(
+        help_text="A string representing the aggregate used in this alert rule."
```

Comment:
what are valid aggregates?

Reply:
The documentation for Create a Metric Alert Rule starts with all the different types of metric alert rules we let you create, which includes the aggregates you need to use for each one! I can add a note here to go back to that section for valid aggregates if we think it's necessary?

Reply:
oh wait i realized i listed the valid values for `datasets` anyway even though they were listed in the different types of metric alert rules. I'll add all the valid aggregates in this section as well!